### DISCUSSION TITLE
Docker builds

### DISCUSSION THREAD

**Discussion by diyoyo** [TriliumNext/Notes#63] [General]
Hi, I just found out today about trilium's announcement and about TriliumNext.
For a while now, I've been wanting to make a real Android app out of it, because the web-based version is clearly not user-friendly on a mobile phone, to be honest.

Anyways, that's not the topic here : I see already more than 280 commits ahead of zadam's repo: are the new docker images available somewhere? Is it worth it to manually build them using the docker file or is it too soon ?

Thanks.

**Comment by meichthys** [comment]
A real native Android app would be amazing!

As for the docker builds, see #66

**Reply by diyoyo** [reply]
I'm starting working on the Android app, but first, I'm making some improvements on the Sender. on the trilium branch. How can I become a contributor on the TriliumNext branch? I'd have one PR for the Sender (adding multiple labels from parsing the hashtags in the title of the note), and I just realized that the api (/src/routes/api/sender.js) is currently not able to handle this, even though the received object is already an Array. So I guess that would be another PR, but for the Notes project.

**Reply by diyoyo** [reply]
Oups, forgive my ignorance. I just went on the TriliumNext Note project and realized this was fixed already in the API.

**Reply by meichthys** [reply]
As for becoming a contributor, I just sent you a TriliumNext member invitation you can accept. 
We also have a `Dev Team` in Github that you can become a part of. Touch base with @eliandoran for more information on that.

**Reply by diyoyo** [reply]
Thanks, what's the difference between being a contributor and the Dev Team?

**Reply by meichthys** [reply]
Teams are mostly just a github way of organizing. It lets you @ the team for notifications, have separate 'team' boards/repos/etc. 
As a contributor you would still be able to create a PR and interact as usual. Currently we have not leaned heavily into teams, but they may be useful in the future. The down-side would be that if we lean-in too heavily, it would make it harder for us to migrate to a different repo host like Gitlab/gitea/etc if we ever wanted to do that.
